devices/dragonboard-410c

```diff
@@ -5,8 +5,12 @@
 {% set rootfs_label = 'rootfs' %}
 {% if DEPLOY_OS is undefined %}{% set DEPLOY_OS = "oe" %}{% endif %}
 {% if BOOT_OS_PROMPT is undefined %}{% set BOOT_OS_PROMPT = "dragonboard-410c:" %}{% endif %}
+{% if TAG_FIRMWARE is undefined %}{% set TAG_FIRMWARE = "old-firmware" %}{% endif %}
 
-{% block device_type %}dragonboard-410c{% endblock %}
+{% block device_type %}dragonboard-410c
+tags:
```

Comment:
maybe tags should be enclosed in their own block? This code looks a bit too hacky.